.github/workflows/fossa.yml

```diff
@@ -0,0 +1,157 @@
+# Name of the GitHub Action workflow
+name: FOSSA Report Generation
+
+# Event triggers for the workflow
+on:
+  workflow_dispatch:  # Run this action manually from the GitHub Actions tab
+     inputs:
+      org:
+        required: false
+        type: choice
+        description: 'Organization to run on'
+        options:
+          - liquibase
+          - datical
+
+  workflow_call:
+    inputs:
+      org:
+        required: false
+        type: string
+        description: 'Organization name: liquibase or datical'
+
+# Define the jobs in the workflow
+jobs:
+  fossa-scan:
+    # Specifies the type of runner to use
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    # Sequence of steps that make up a single job
+    steps:
+      # Checkout the code to the GitHub runner
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK for Datical
+        if: inputs.org == 'datical'
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'maven'
+          server-id: datical
+          server-username: REPO_MAVEN_USER
+          server-password: REPO_MAVEN_PASSWORD
+
+      - name: Log inputs
+        run: |
+          echo "Org: ${{ inputs.org }}"
+
+      - name: Set up JDK for Liquibase
+        if: inputs.org == 'liquibase'
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+            #look for dependencies in maven
+      - name: maven-settings-xml-action for Liquibase
+        if: inputs.org == 'liquibase'
+        uses: whelk-io/maven-settings-xml-action@v22
+        with:
+          repositories: |
```

Comment:
Hi @sayaliM0412 - looks like there are liquibase-pro and liquibase (foundation) repos, which are not related to Enterprise. Is this PR needed for DAT-18147? If it is needed, then do we need liquibase-pro and liquibase (foundation) repos part?